api/v1beta1/azureclusteridentity_types.go

```diff
@@ -43,7 +43,7 @@ type AllowedNamespaces struct {
 
 // AzureClusterIdentitySpec defines the parameters that are used to create an AzureIdentity.
 type AzureClusterIdentitySpec struct {
-	// UserAssignedMSI or Service Principal
```

Comment:
should we clean up all these comments so that the property name is the first string in the comment, e.g.
```
// Type is the identity type.
// Valid options are ServicePrincipal, ServicePrincipalCertificate, or ManualServicePrincipal.
Type IdentityType `json:"type"`
// ResourceID is the user-assigned MSI resource id.
// +optional
ResourceID string `json:"resourceID,omitempty"`
```

etc

Reply:
done